Aztec/Classes/Extensions/NSAttributedString+Attachments.swift

```diff
@@ -2,6 +2,15 @@ import Foundation
 import UIKit
 
 
+// MARK: - Constants
+//
+extension String {
+    /// String containing the NSTextAttachment Character
+    ///
+    static let textAttachment = String(UnicodeScalar(NSAttachmentCharacter)!)
```

Comment:
Loved this little touch!

Reply:
I absolutely LOVE 'extensible enums' (that's this trick's name!)